builtin/logical/pki/path_config_ca.go

```diff
@@ -21,6 +22,25 @@ secret key and certificate.`,
 		Operations: map[logical.Operation]framework.OperationHandler{
 			logical.UpdateOperation: &framework.PathOperation{
 				Callback: b.pathImportIssuers,
+				Responses: map[int][]framework.Response{
+					http.StatusOK: {{
+						Description: "OK",
+						Fields: map[string]*framework.FieldSchema{
+							"mapping": {
+								Type:        framework.TypeMap,
+								Description: "A mapping of which keys belong to which issuers",
+							},
+							"imported_keys": {
+								Type:        framework.TypeCommaStringSlice,
+								Description: "",
```

Comment:
@cipherboy Do you have a good description of these per chance?

Reply:
Yeah! I like the ones in the [docs](https://developer.hashicorp.com/vault/api-docs/secret/pki#import-ca-certificates-and-keys): 

> The response will indicate what issuers and keys were created as part of this request (in the `imported_issuers` and `imported_keys`), along with a `mapping` field, indicating which keys belong to which issuers (including from already imported entries present in the same bundle).

So perhaps something like:

 - `imported_keys`: net-new keys imported as part of this request
 - `imported_issuers`: net-new issuers imported as part of this request
 - `mapping`: mapping of issuer_id->key_id for all issuers included in this request

?

Reply:
Perfect! That works for me

Reply:
Thanks! I updated it